src/patternfly/components/Drawer/drawer-header.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-drawer__header{{#if drawer-header--modifier}} {{drawer-header--modifier}}{{/if}}"
```

Comment:
What is this element used for?

Reply:
It's part of the master/detail layout concept. [This example](https://patternfly-next-pr-2645.surge.sh/documentation/core/demos/masterdetail/master-detail-card-view-expanded) uses it. 

https://marvelapp.com/9iefh5b/screen/63567619

Reply:
Ah, does that need to be a part of the drawer component though? It looks kinda random. I would expect the space between the text and paragraph in that demo to come from something like a stack layout or a content component that wraps the header text and paragraph and the header text is some sort of heading element.

Reply:
Doesn't exist anymore. `drawer__body` goes directly in `drawer__content` and `drawer__panel` and replaces `drawer__header`.